test/cqlpy/test_filtering.py

```diff
@@ -3,7 +3,8 @@
 # SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
 
 #############################################################################
-# Tests for the SELECT requests with various filtering expressions.
+# Tests originally for the SELECT requests with various filtering expressions,
+# later extended with other CQL commands supporting WHERE's conditional expressions.
```

Comment:
I don't think your new test belongs here. Your new test checks the "IF" syntax of LWT, so I think it belongs in test_lwt.py.